config/core/resources/sinkbindings.yaml

```diff
@@ -25,43 +25,48 @@ metadata:
 spec:
   group: sources.knative.dev
   versions:
-  - &version
```

Comment:
What's the difference between new yaml and old yaml? Is this just format change?

Reply:
is just format plus this:

  - <<: *version
      name: v1beta1
      served: true
      storage: false